packages/teleport-plugin-html-base-component/src/node-handlers.ts

```diff
@@ -657,14 +659,34 @@ const generateComponentContent = async (
       }
     }
 
+    if (attribute?.type === 'expr') {
+      const [ctxId, ...refPath] = attribute.content.split('?.')
```

Comment:
little catch, if these expressions are added by end users like this `item.name` instead of `item?.name`. There might be a case it fails. But, i don' think thats the case right. These expressions are ususally auto generated by builder. So, It should be fine.